packages/sdk/src/tests/Fuel.test.ts

```diff
@@ -75,6 +81,34 @@ describe('Fuel', () => {
     expect(isAdded).toEqual(true);
   });
 
+  test('addAbi', async () => {
+    const abiMap = {
+      [AbiContractId]: FlatAbi,
+    };
+    const isAdded = await fuel.addAbi(abiMap);
+    expect(isAdded).toEqual(true);
+  });
+
+  test('getAbi', async () => {
+    const abiMap = {
+      [AbiContractId]: FlatAbi,
+    };
+    await fuel.addAbi(abiMap);
+    const abi = await fuel.getAbi(AbiContractId);
+
+    expect(abi).toStrictEqual(FlatAbi);
```

Comment:
is there a reason we check equality against the FlatAbi and not the abiMap?

Reply:
yes. the abiMap is a map of { contractId: flatAbi }..